content/SCALE/SCALECLIReference/Account/CLIgroup.md

```diff
@@ -59,53 +71,28 @@ Where *3022* is the group id number, *TestGroup* is the group name, *false* is a
 {{< /expand >}}
 {{< /expand >}}
 
-### Create Interactive Arguments Editor
-
-The `create --` option opens an **interactive arguments editor**.
-
-{{< expand "Using the Create Interactive Arguments Editor" "v" >}}
-{{< trueimage src="/images/SCALE/CLI/Account/GroupCreateInteractiveArgumentsEditor.png" alt="Group Create Interactive Arguments Editor" id="1: Group Create Interactive Arguments Editor" >}}
-
-The interactive arguments editor provides a text user interface (TUI) where you can configure properties and options. The TUI also provides some information on required properties, defaults, and expected input types (string, boolean, integer, or array).
-
-In the TUI, most properties are initially marked as comments with the `#` symbol, indicating that they are not yet configured. However, `name:` is shown as a required field.
-
-To provide values for the other properties, you need to remove the `#` comment designator from the corresponding line in the TUI.
-
-{{< hint type=tip >}}
-A group name is the only required value for group creation. If a group identification number is not provided, it is automatically filled with the next one available. All other properties are optional.
-
-See the relevant rows in [Configuration Properties](#create-configuration-properties) for more information.
-{{< /hint >}}
-
-A space is required between the provided properties and entered values, for example `name: test_group`.
-
-Press <kbd>F2</kbd> or click **Save** to save the modified file.
-
-Press <kbd>F10</kbd>, <kbd>Esc</kbd>, or click **Quit** to exit the TUI.
-The `create` command automatically executes upon exit.
-{{< /expand >}}
-
-### Create Configuration Properties
+#### Create Configuration Properties
 
-`create` has seven available properties for group configuration. They are `gid`, `name`, `smb`, `sudo_commands`, `sudo_commands_nopasswd`, `allow_duplicate_gid`, and `users`. See the table below for more details.
+`create` has seven available properties for group configuration. They are `gid`, `name`, `smb`, `sudo_commands`, `sudo_commands_nopasswd`, `allow_duplicate_gid`, and `users`.
```

Comment:
Recommend adding "For more information on these properties see "Property Functions and Examples" section below.
Also recommend you mention here the number of required properties, as in
```suggestion
`create` has one required and six optional properties for group configuration. `name` is required. 
The optional properties are `gid`,  `smb`, `sudo_commands`, `sudo_commands_nopasswd`, `allow_duplicate_gid`, and `users`.
```